packages/venia-ui/lib/components/Header/cartTrigger.css

```diff
@@ -17,3 +37,16 @@
     composes: root from '../Icon/icon.css';
     --fill: rgb(var(--venia-text));
 }
+
+/*
+ * Mobile-specific styles.
+ */
+@media (max-width: 960px) {
+    .root_desktop {
+        display: none;
+    }
+
+    .root_mobile {
+        display: block;
```

Comment:
On mobile the cart trigger is FUNKY

[![Image from Gyazo](https://i.gyazo.com/f4bddc50221957082f892e14cefa2edb.png)](https://gyazo.com/f4bddc50221957082f892e14cefa2edb)

Reply:
Yeah. The grid container seems to be missing `grid-auto-flow: column` or similar.

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2494/commits/29cb5bef7ad1a0376bd8d7b418eadbb7dac3a7a7.